sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyAlertConfigurationLiveTests.cs

```diff
@@ -205,7 +205,8 @@ public async Task CreateAndGetAlertConfigurationWithOptionalSingleMetricConfigur
                         UpperBound = 20.0,
                         LowerBound = 10.0,
                         CompanionMetricId = metricId,
-                        ShouldAlertIfDataPointMissing = true
+                        ShouldAlertIfDataPointMissing = true,
+                        MeasureType = BoundaryMeasureType.Mean
```

Comment:
For most tests, we'll leave the `MeasureType` property untouched. We'll just check that its default value (`BoundaryMeasureType.Value`) has been set.

We have one "Create-operation" test (this one) that sets the property to `Mean`, and one "Update-operation" test that updates the property to `Mean`.